pkg/yurthub/poolcoordinator/coordinator.go

```diff
@@ -231,7 +240,37 @@ func (coordinator *coordinator) Run() {
 			}
 			metrics.Metrics.ObservePoolCoordinatorYurthubRole(electorStatus)
 
-			switch electorStatus {
+			if coordinator.cnt == math.MaxUint64 {
+				// cnt will overflow, reset it.
+				coordinator.cnt = 0
+				// if statusInfoChan channel also has data, clean it.
+				length := len(coordinator.statusInfoChan)
+				if length > 0 {
+					i := 0
+					for v := range coordinator.statusInfoChan {
+						klog.Infof("clean statusInfo data %+v when coordinator.cnt is reset", v)
+						i++
+						if i == length {
+							break
+						}
+					}
+				}
+			}
+			coordinator.cnt++
```

Comment:
Just a small bug(even not a bug currently).
If coordinator.cnt == math.MaxUint64, it will be firstly set as 0, and then self increase here. The cnt of final enqueue status is 1, 0 is skipped.